scripts/heroku_sync_db_to_local.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/zsh
+# Script for syncing remote DB to local
+
+APP_NAME=$1
```

Comment:
Could we get the app name from the current directory, like heroku does? We could still have $APP_NAME as optional parameter.